Sources/FileCheck/FileCheck.swift

```diff
@@ -110,7 +110,7 @@ extension FileCheckSource: ExpressibleByStringLiteral {
 /// - parameter checkNot: Specifies zero or more prefixes to implicitly reject
 ///   in the output stream.  This can be used to implement LLVM-verifier-like
 ///   checks.
-/// - parameter file: The file to check against.  Defaults to the file that
+/// - parameter against: The source of Check Strings to check against.  Defaults to the file that
```

Comment:
the argument is `source`. `against` is the parameter label.

Reply:
oh yes, indeed. Thanks for the remark, fixed in [7a6517d](https://github.com/llvm-swift/FileCheck/pull/27/commits/7a6517d54dd62fb253fd01ce06c50e72247820b3).

Reply:
thanks!